x-pack/plugins/stack_alerts/server/alert_types/geo_containment/index.ts

```diff
@@ -15,5 +15,5 @@ interface RegisterParams {
 
 export function register(params: RegisterParams) {
   const { logger, alerts } = params;
-  alerts.registerType(getAlertType(logger));
+  alerts.registerType((getAlertType(logger) as unknown) as AlertType);
```

Comment:
Doing `as unknown) as AlertType` throughout would remove the TS type safety we have for alert type. I wonder what caused this to break?